R/navbar-menu.R

```diff
@@ -15,6 +21,10 @@ menu_link <- function(text, href, target = NULL) {
 menu_links <- function(text, href) {
   purrr::map2(text, href, menu_link)
 }
+menu_theme <- function(text, icon, theme) {
+  purrr::compact(list(text = text, href = "#", theme = theme, icon = icon))
```

Comment:
For nested pages, e.g. in articles, pkgdown rewrites this to `href="../#"` which ends up navigating you away from the page. It'd be better if pkgdown could use a `<button>` element instead of an `<a>` element for these menu items.